### ISSUE TITLE
[Torch] Add support for Huber Loss function

### ISSUE THREAD

**Issue by zahidwx** [llvm/torch-mlir#4247]
### Huber Loss function
https://docs.pytorch.org/docs/stable/generated/torch.nn.HuberLoss.html#torch.nn.HuberLoss

This op is missing in torch dialect. It can be implemented via decomposition to other primitive torch ops.
